app/views/lectures/show.html.erb

```diff
@@ -56,15 +58,15 @@
     <% if @lecture.questions_enabled %>
       <%= react_component("QuestionsApp",  {user_id: @current_user.id, is_student: @current_user.is_student, lecture_id: @lecture.id, course_id: @lecture.course.id, interactions_enabled: @lecture.allow_interactions?, questions_list: Question.questions_for_lecture(@lecture, current_user)} ) %>
     <% else %>
-      <p class="h1"> Questions are not enabled. :(</p>
```

Comment:
This one seems a bit redundant. We don't show the tab if questions are not enabled so I'd say we don't need a fallback... I mean, it doesn't hurt but seems useless. Same for polls and feedback... On the other hand I think it's not the best UX practice to really hide a feature that was once there...

Reply:
Except of this Code looks good. @adrianjost I think you can decide whether this is good UX or not.

- [ ] Check if good UX, if not delete the fallback and make the feature completely hidden.

Reply:
But it is also bad UX to show features that you can not use and fill up the screen on small mobile devices with that.

I kept the fallbacks for two reasons:
a) if we ever want to go back because of yet unknown reasons its easier
b) we can better test and debug that a feature is actually disabled.